docs/getting-started/integrating-codacy-with-your-git-workflow.md

```diff
@@ -0,0 +1,61 @@
+---
+description: Configure Codacy to block merging pull requests that don't meet your quality standards.
+---
+
+# Integrating Codacy with your Git workflow
+
+Once you've [customized your setup](customizing-your-setup.md) to best match your needs, you can integrate Codacy with your Git workflow to display analysis results and code coverage on your Git provider and block problematic pull requests.
```

Comment:
Make the intro more assertive, then reuse bits of the intro on "Next steps" of the previous page. :wink: